docs/content/tutorial/connecting-to-external-services.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: "Tutorial, part seven: Connecting to external services | Dagster Docs"
+description: Learn how to connect to external services in Dagster with resources
+---
+
+Throughout the tutorial, you have built a data pipeline that connects to different external services, such as Hacker News. By building this asset graph, Dagster acts as a control plane of multiple different external services. However, connecting to these services and managing their use can be difficult. Resources are the recommended way to manage these connections in Dagster.
+
+Resources are Dagster objects that connect to external services, such as Slack, Snowflake, or S3. In this section, you’ll learn more about resources, including how to:
```

Comment:
```suggestion
Resources are Dagster objects that connect to external services, such as Slack, Snowflake, or Amazon S3. In this section, you’ll learn more about resources, including how to:
```

⛏️ The first time a product like this is mentioned, the name of the company/provider/etc should be included. After, it can just be the product - ex: `S3`